client/src/pages/lessons/edit.tsx

```diff
@@ -181,6 +206,47 @@ export interface LessonEditSearch {
   copyLesson?: string;
 }
 
+declare module "@mui/material/styles" {
+  interface Palette {
+    primary: Palette["primary"];
+    secondary: Palette["primary"];
+    error: Palette["primary"];
+    warning: Palette["primary"];
+    success: Palette["primary"];
+  }
+
+  interface PaletteOptions {
+    primary?: PaletteOptions["primary"];
+    secondary?: PaletteOptions["primary"];
+    error?: PaletteOptions["primary"];
+    warning?: PaletteOptions["primary"];
+    success?: PaletteOptions["primary"];
+  }
+}
+
+const buttonTheme = createTheme({
+  palette: {
+    primary: {
+      main: "#0C60AD",
+    },
+    secondary: {
+      main: "#000000",
+    },
+    warning: {
+      main: "#FFFF00",
+      contrastText: "#000000",
+    },
+    success: {
+      main: "#008000",
+    },
+    error: {
+      main: "#FF0000",
+    },
+  },
+});
+
+const options = ["No Media", "Add Image", "Add Video"];
+
```

Comment:
A lot of CSS that I don't have enough expertise to review.  Best to get a review from either Aaron, or Dilan.  Maybe both for the initial one.